src/System.Memory/tests/ReadOnlySpan/IsWhiteSpace.cs

```diff
@@ -0,0 +1,105 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.SpanTests
+{
+    public static partial class ReadOnlySpanTests
+    {
+        private static readonly char[] s_whiteSpaceCharacters = { (char)9, (char)10, (char)11, (char)12, (char)13, (char)32, (char)133, (char)160, (char)5760 };
```

Comment:
nit: instead of doing (char)9, would be better to write it as '\u0009' note this is hex number